From: Simon McVittie <smcv@debian.org>
Date: Tue, 17 Jan 2017 23:50:55 +0000
Subject: tests: Install testpython.py executable

The test that uses this file executes it. Install it as an extra
script rather than data to ensure that it can work.
---
 tests/Makefile.am.inc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/Makefile.am.inc b/tests/Makefile.am.inc
index d1b4a43..4c220ab 100644
--- a/tests/Makefile.am.inc
+++ b/tests/Makefile.am.inc
@@ -79,6 +79,7 @@ dist_installed_test_extra_scripts += \
 	tests/make-test-app.sh \
 	tests/make-test-runtime.sh \
 	tests/make-test-bundles.sh \
+	tests/testpython.py \
 	$(NULL)
 
 dist_installed_test_data = \
@@ -89,7 +90,6 @@ dist_installed_test_data = \
 	tests/session.conf.in \
 	tests/0001-Add-test-logo.patch \
 	tests/org.test.Python.json \
-	tests/testpython.py \
 	tests/importme.py \
 	tests/importme2.py \
 	$(NULL)
